File: sum_vectors.lua
a = {}
b = {}

for i = 1, 10000000 do
    a[i] = i
    b[i] = i
end

for i = 1, #a do
    a[i] = a[i] + b[i]
end

print("last element = " .. a[#a])

File: sum_vectors.js
let a = Array.from({ length: 10_000_000 }, (_, i) => i + 1);
const b = Array.from({ length: 10_000_000 }, (_, i) => i + 1);

for (let i = 0; i < a.length; ++i) {
  a[i] += b[i];
}

console.log(`last element = ${a[a.length - 1]}`);
